### PR TITLE
feat: Add Azure and Oracle Fabric Cloud Router connections

### PR DESCRIPTION
**Pull request by manu-equinix**
Adds the below Fabric cloud router(FCR) connections types and related tests and examples

- FCR to Azure cloud
- FCR to Oracle cloud

### CODE REVIEW

File: examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/two-connections/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# ECX Fabric Layer2 Single Connection from fabric cloud router to Azure
```
Comment (thogarty) [inline examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/two-connections/README.md:1]: This README.md needs to be updated. I missed it when adding the two-connections example. I've submitted a PR on your fork and branch for this to be updated: https://github.com/manu-equinix/terraform-provider-equinix/pull/13

Other than that everything looks good to me 👍

**Review by thogarty — DISMISSED** [review]

File: examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/two-connections/README.md (hunk 1 above)
Comment (manu-equinix) [reply examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/two-connections/README.md]: Merged

File: examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/two-connections/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,20 @@
+equinix_client_id = "Equinix_Client_Id"
```
Comment (ctreatma) [inline examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/two-connections/terraform.tfvars:1]: We talked about ignoring `.tfvars` files and (optionally) including a `.tfvars.example`.  Is that coming later?

**Review by ctreatma — APPROVED** [review]

File: examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/two-connections/terraform.tfvars (hunk 2 above)
Comment (manu-equinix) [reply examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/two-connections/terraform.tfvars:1]: @ctreatma Yes, Tim has a PR ready for that and we plan to merge in next few days.

File: equinix/fabric_connection_schema.go [hunk 3]
```diff
@@ -453,6 +453,7 @@ func createRedundancySch() map[string]*schema.Schema {
 		"group": {
 			Type:        schema.TypeString,
 			Computed:    true,
+			Optional:    true,
```
Comment (displague) [inline equinix/fabric_connection_schema.go:456]: The docs will need to be updated with this change: https://github.com/equinix/terraform-provider-equinix/blob/main/docs/resources/equinix_fabric_connection.md#nested-schema-for-a_sideaccess_pointportredundancy

File: examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/single-connection/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# ECX Fabric Layer2 Single Connection from fabric cloud router to Azure
```
Comment (displague) [inline examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/single-connection/README.md:1]: In the future, I'd like us to consider an alternative approach to providing examples: https://github.com/equinix/terraform-provider-equinix/issues/384

Comment (thogarty) [reply examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/single-connection/README.md:1]: I like the sound of this 👍

File: equinix/fabric_connection_schema.go (hunk 3 above)
Comment (thogarty) [reply equinix/fabric_connection_schema.go:456]: Updates pending PR Approval by @manu-equinix : https://github.com/manu-equinix/terraform-provider-equinix/pull/14

**Review by thogarty — APPROVED** [review]
LGTM

File: equinix/fabric_connection_schema.go (hunk 3 above)
Comment (manu-equinix) [reply equinix/fabric_connection_schema.go:456]: @displague @thogarty This has been resolved(merged). Please re-approve.

Comment (thogarty) [reply equinix/fabric_connection_schema.go:456]: @manu-equinix approval given 👍

**Review by ctreatma — APPROVED** [review]

**Pull request merged**